server/index.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors')
+
+const mysql = require('mysql');
+
+
+const app = express();
+
+//Middlewares
+app.use(bodyParser.json());
+app.use(cors({
+  origin: '*'
+}));
+
+
+const PORT = process.env.PORT || 3500;
+
+const connection = mysql.createConnection({
```

Comment:
Maybe use env variables for the DB connection, @GrenderG should point or setup the right ones